mslib/msui/msui_mainwindow.py

```diff
@@ -468,6 +473,146 @@ def __init__(self, mscolab_data_dir=None, *args):
             self.actionUpdater.triggered.connect(self.updater.show)
         self.openOperationsGb.hide()
 
+        self.last_changed_item = None
+        self.StoreLayout.clicked.connect(self.store_operation_data)
+        self.RestoreLayout.clicked.connect(self.restore_operation_data)
+        self.allowed_widget_names = ["TopViewWindow", "SideViewWindow", "TableViewWindow", "LinearWindow"]
+        self.listOperationsMSC.itemActivated.connect(self.operation_changed)
+
+    def operation_changed(self):
+        selected_items = self.listOperationsMSC.selectedItems()
+
+        if not selected_items:
+            return
+
+        selected_item = selected_items[0]
+        operation_name = selected_item.text()
+        loaded_data = requests.get(f"{self.mscolab.mscolab_server_url}/load_operation_layout", params={"operation_name": operation_name})
+        if loaded_data.text == "None":
```

Comment:
An older version of the mscolab server returns something like 404. You should catch this and treat it as None for better compatability.